tests/CSharp/CSharp.h

```diff
@@ -1193,3 +1193,14 @@ struct HasComplexArray
 {
     ComplexArrayElement complexArray[ARRAY_LENGTH_MACRO];
 };
+
+class DLL_API TestIndexedProperties
+{
+public:
+	int operator[](const int& key);
```

Comment:
You have used a tab here while we use 4 spaces in C++ too. You could change your VS settings in order not to fix it manually each time.